### PR TITLE
Fix build icons workflow

### PR DESCRIPTION
**Pull request by lunatic-fox**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened

## Features

**This PR closes NONE**

## Notes
Skips `imgur_step` and removes the lines related to this step on `release_message_step`. It also uses the last version of Windows, GitHub Checkout, Setup Python and Python on 3.10 version.

### CODE REVIEW

File: .github/workflows/build_icons.yml [hunk 1]
```diff
@@ -44,15 +44,15 @@ jobs:
...
-          client_id: ${{secrets.IMGUR_CLIENT_ID}}
+      # - name: Upload screenshot of the newly made icons
+      #   id: imgur_step
+      #   uses: devicons/public-upload-to-imgur@v2.2.2
+      #   if: success()
+      #   with:
+      #     # will have "new_icons.png" and "new_svgs.png"
+      #     # in that order (cause sorted alphabetically)
+      #     path: ./screenshots/*.png
+      #     client_id: ${{secrets.IMGUR_CLIENT_ID}}
```
Comment (lunatic-fox) [inline .github/workflows/build_icons.yml:55]: This part is skipped by comment, but it can be removed.

Comment (Panquesito7) [reply .github/workflows/build_icons.yml:55]: Let's keep it as-is so that we can fix it in the future. 👍

**Review by Panquesito7 — COMMENTED** [review]
Are you sure we do not need to update other workflows to use newer versions, as done in #1731?

**Comment by lunatic-fox** [comment]
> Are you sure we do not need to update other workflows to use newer versions, as done in #1731?

Well, I do think so, since this action uses geckodriver on a Windows machine and on `develop` branch geckodriver is on an Ubuntu machine.

File: .github/workflows/build_icons.yml [hunk 2]
```diff
@@ -3,13 +3,13 @@ on: workflow_dispatch
 jobs:
   build:
     name: Get Fonts From Icomoon
-    runs-on: windows-2019
+    runs-on: windows-latest
```
Comment (lunatic-fox) [inline .github/workflows/build_icons.yml:6]:
```suggestion
    runs-on: ubuntu-latest
```

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🚀
Let's test it out.

**Pull request merged**